public/app/features/explore/RichHistory/RichHistoryQueriesTab.tsx

```diff
@@ -9,8 +9,8 @@ import { RichHistoryQuery, ExploreId } from 'app/types/explore';
 import { stylesFactory, useTheme } from '@grafana/ui';
```

Comment:
Change of SortOrder import as we have moved LogsSortOrder to grafana/data and moved SortOrder for Rich history to app/core/utils/richHistory.